crates/sui-framework/sources/coin.move

```diff
@@ -249,6 +310,14 @@ module sui::coin {
         balance::increase_supply(&mut cap.total_supply, value)
     }
 
+    spec mint_balance {
```

Comment:
The specs here and for `mint` seem very similar. Maybe use `schema`? https://github.com/move-language/move/blob/main/language/move-prover/doc/user/spec-lang.md#schemas
Same for `burn` and `burn_`.

Reply:
Does it help if you add `<T>` to the schema signature too? Like
```
spec schema MintSpecs<T> {
    cap: TreasuryCap<T>;
    ...
}
```
There are some examples in Diem Framework specs that might be helpful: https://github.com/move-language/move/blob/main/language/documentation/examples/diem-framework/move-packages/DPN/sources/Diem.move#L453

Reply:
Yeah, I found them :-) Hence my comment deletion :-) Getting there!

Reply:
Done!